codespell_lib/data/dictionary.txt

```diff
@@ -2835,6 +2881,10 @@ breefly->briefly
 brefore->before
 breif->brief
 breifly->briefly
+bresh->brush
+breshed->brushed
+breshes->brushes
+breshing->brushing
```

Comment:
fresh
freshd
freshes
freshing

Reply:
i appreciate the feedback, but this one is kind of a stretch.  
I'd green-light: `bresh->brush, fresh,`  
but for freshed, freshes, freshing...which are obsolete...I'd red light that

Reply:
@peternewman please weigh-in, thanks